subgraphs/gmx/src/utils/constants.ts

```diff
@@ -0,0 +1,184 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+////////////////////
+///// Versions /////
+////////////////////
+
+export const PROTOCOL_SCHEMA_VERSION = "1.0.2";
```

Comment:
remove the versions here, and you can add to `deployments/deployments.json` instead.

And if you are not already using `messari-subgraph-cli` npm package that will make it much easier to build and deploy your subgraph for testing. Lmk if you have questions ab it, there are docs on how it works and some nice videos Steege has made

Reply:
Fixed at b5d5ed81355ebf1ff84a231c0fa5aba7df61acf0 Thanks!